packages/noports_core/test/sshnp/util/sshrvd_channel/sshrvd_channel_test.dart

```diff
@@ -50,25 +53,8 @@ void main() {
         params: mockParams,
         sessionId: sessionId,
         sshrvGenerator: sshrvGeneratorStub,
-        notify: (_, __) async {
-          final testIp = '123.123.123.123';
-          final portA = 10456;
-          final portB = 10789;
-
-          notificationStreamController.add(
-            AtNotification.empty()
-              ..id = Uuid().v4()
-              ..key = '$sessionId.${Sshrvd.namespace}'
-              ..from = '@sshrvd'
-              ..to = '@client'
-              ..epochMillis = DateTime.now().millisecondsSinceEpoch
-              ..value = '$testIp,$portA,$portB',
-          );
-          notifyStub();
-        },
-        subscribe: ({regex, shouldDecrypt = false}) {
-          return subscribeStub();
-        },
```

Comment:
This removed function is now in a when(() => notifyStub).thenAnswer(...) block later on